meteor/sofie-server/model/entities/segment.ts

```diff
@@ -82,7 +82,7 @@ export class Segment {
 		}
 		return part
 	}
-
+	// TODO: Verify if a check to return early should be added, in case array is empty/only contains 1 element
```

Comment:
Did you run into issues with this? `.sort()` is a method of `Array` and should handle this already.

Reply:
I think it was just something that i noted, at a quick glance.
You are propably right that the `.sort()` handles it already.

Reply:
Todo Comment has been removed.

Reply:
Methods on arrays like `.map()`, `.foreach()`, `.filter()` etc already handles that the array is empty and so forth. It would be extremely weird if `.sort()` behaved any differently.

This snippet will log nothing:
```
const someArray: string[] = []
someArray.foreach(o => console.log(o))
```